src/vuestic-theme/vuestic-components/vuestic-collapse/VaCollapse.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="va-collapse">
+    <div
+      class="va-collapse__header"
+      :class="{
+        'vuestic-collapse__header--no-header': noHeader,
+      }"
+      @click="onHeaderClick()"
+    >
+      <template v-if="noHeader">
+        <slot name="header"/>
+      </template>
+      <div
+        v-else
+        class="va-collapse__header__content va-row "
+      >
+        <slot name="header"/>
+        <i
+          v-if="show"
+          class="fa fa-angle-up va-collapse__header__icon"
+        />
+        <i
+          v-else
+          class="fa fa-angle-down va-collapse__header__icon"
+        />
+      </div>
+    </div>
+    <div
+      class="va-collapse__body"
+      :class="{'va-collapse__body--with-background': isBackgroundExists}"
```

Comment:
Use `.va-collapse--with-background > .va-collapse__body` instead. Also keep margins the same for both modes, just change background color.